packages/framework/ai-collab/src/test/explicit-strategy/agentEditingReducer.spec.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
```

Comment:
Not a fan of disabling lint rules globally like this without a good explanation. Were there so many instances of it? Also, I think this needs to go under the header so policy checks pass.